tests/test-skeleton/sympy-0.7.5/meta.yaml

```diff
@@ -4,7 +4,7 @@ package:
 
 source:
   fn: sympy-0.7.5.tar.gz
-  url: https://pypi.python.org/packages/source/s/sympy/sympy-0.7.5.tar.gz
+  url: https://pypi.python.org/packages/8c/a5/5fa8adee81837687f7315122769fc0b0e8b042c69e2fe5809c41191c7183/sympy-0.7.5.tar.gz
```

Comment:
Unrelated, but it is here. Why do we now have to do this with the URLs?

I have seen this is needed on conda-forge too. Would be nice if we could handle a redirect or whatever so that we can avoid this ugly URL somehow.

cc @ocefpaf